### PR TITLE
Changed the regexp that pass trough any log message for keep in ```[c…

### PR DESCRIPTION
**Pull request by arturmamedov**
Keep in `[coontext]` and `{extra data}` to, and added extra to view.

### CODE REVIEW

**Comment by arturmamedov** [comment]
This work for the default monolog format
```
"[%datetime%] %channel%.%level_name%: %message% %context% %extra%\n"
```

one thing is that i not undersant from where you get ```'stack'``` that you use after for open close the information about log message.

**Comment by rap2hpoutre** [comment]
Ok thank you! Does it work with a fresh install of laravel 4.2 and 5.*? (let's say 5.2 is enough for your tests)

File: src/Rap2hpoutre/LaravelLogViewer/LaravelLogViewer.php [hunk 1]
```diff
@@ -128,6 +128,8 @@ public static function all()
                             'date' => $current[1],
                             'text' => $current[2],
                             'in_file' => isset($current[3]) ? $current[3] : null,
+							'context' => isset($current[4]) ? $current[4] : null,
```
Comment (rap2hpoutre) [inline src/Rap2hpoutre/LaravelLogViewer/LaravelLogViewer.php]: Use spaces (not tab) please

File: src/views/log.blade.php [hunk 2]
```diff
@@ -80,7 +80,13 @@
     @if ($log['stack']) <a class="pull-right expand btn btn-default btn-xs" data-display="stack{{{$key}}}"><span class="glyphicon glyphicon-search"></span></a>@endif
     {{{$log['text']}}}
     @if (isset($log['in_file'])) <br />{{{$log['in_file']}}}@endif
-    @if ($log['stack']) <div class="stack" id="stack{{{$key}}}" style="display: none; white-space: pre-wrap;">{{{ trim($log['stack']) }}}</div>@endif
+    @if ($log['stack'] || $log['extra'])
+	  <div class="stack" id="stack{{{$key}}}" style="display: none; white-space: pre-wrap;">{{{ trim($log['stack']) }}}
```
Comment (rap2hpoutre) [inline src/views/log.blade.php]: Use 2 spaces (not tab) please

**Comment by arturmamedov** [comment]
Yes it work with Laravel 5.1.35 in my case, and it has a fallback if no extra options is used in monolog. I adjust the indentation and check in Laravel 4.2 and the new 5.3.x

**Pull request closed without merging**